uint64 timestamp			# time since system start (microseconds)

uint8 INDEX_ROLL = 0
uint8 INDEX_PITCH = 1
uint8 INDEX_YAW = 2
uint8 INDEX_THROTTLE = 3

float32[4] control_power

# TOPICS actuator_controls_status actuator_controls_status_0 actuator_controls_status_1
